.packit.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
+      rhel-8:
         additional_modules: "foreman:el8"
```

Comment:
Should we use the devel module? Not sure if it matters here since we don't need NodeJS